tsconfig.json

```diff
@@ -1,146 +1,170 @@
-      ],
-      "@theia/workspace/lib/*": [
-        "packages/workspace/src/*"
-      ],
-      "@theia/terminal/lib/*": [
-        "packages/terminal/src/*"
-      ],
-      "@theia/languages/lib/*": [
-        "packages/languages/src/*"
-      ],
-      "@theia/editor/lib/*": [
-        "packages/editor/src/*"
-      ],
-      "@theia/mini-browser/lib/*": [
-        "packages/mini-browser/src/*"
-      ],
-      "@theia/monaco/lib/*": [
-        "packages/monaco/src/*"
-      ],
-      "@theia/java/lib/*": [
-        "packages/java/src/*"
-      ],
-      "@theia/java-debug/lib/*": [
-        "packages/java-debug/src/*"
-      ],
-      "@theia/debug/lib/*": [
-        "packages/debug/src/*"
-      ],
-      "@theia/debug-nodejs/lib/*": [
-        "packages/debug-nodejs/src/*"
-      ],
-      "@theia/python/lib/*": [
-        "packages/python/src/*"
-      ],
-      "@theia/git/lib/*": [
-        "packages/git/src/*"
-      ],
-      "@theia/process/lib*": [
-        "packages/process/src/*"
-      ],
-      "@theia/markers/lib/*": [
-        "packages/markers/src/*"
-      ],
-      "@theia/outline-view/lib/*": [
-        "packages/outline-view/src/*"
-      ],
-      "@theia/messages/lib/*": [
-        "packages/messages/src/*"
-      ],
-      "@theia/merge-conflicts/lib/*": [
-        "packages/merge-conflicts/src/*"
-      ],
-      "@theia/output/lib/*": [
-        "packages/output/src/*"
-      ],
-      "@theia/callhierarchy/lib/*": [
-        "packages/callhierarchy/src/*"
-      ],
-      "@theia/typehierarchy/lib/*": [
-        "packages/typehierarchy/src/*"
-      ],
-      "@theia/variable-resolver/lib/*": [
-        "packages/variable-resolver/src/*"
-      ],
-      "@theia/editorconfig/lib/*": [
-        "packages/editorconfig/src/*"
-      ],
-      "@theia/console/lib/*": [
-        "packages/console/src/*"
-      ],
-      "@theia/scm/lib/*": [
-        "packages/scm/src/*"
-      ],
-      "@theia/getting-started/lib/*": [
-        "packages/getting-started/src/*"
-      ],
-      "@theia/file-search/lib/*": [
-        "packages/file-search/src/*"
-      ],
-      "@theia/plugin-dev/lib/*": [
-        "packages/plugin-dev/src/*"
-      ],
-      "@theia/plugin-ext/lib/*": [
-        "packages/plugin-ext/src/*"
-      ],
-      "@theia/plugin-ext-vscode/lib/*": [
-        "packages/plugin-ext-vscode/src/*"
-      ],
-      "@theia/plugin-metrics/lib/*": [
-        "packages/plugin-metrics/src/*"
-      ],
-      "@theia/userstorage/lib/*": [
-        "packages/userstorage/src/*"
-      ],
-      "@theia/task/lib/*": [
-        "packages/task/src/*"
-      ],
-      "@theia/api-samples/lib/*": [
-        "examples/api-samples/src/*"
-      ]
-    },
+    "composite": true,
     "plugins": [
       {
         "name": "tslint-language-service"
       }
     ]
   },
-  "include": [
-    "dev-packages/*/src",
-    "packages/*/src",
-    "examples/*/src",
-    "examples/*/test"
+  "include": [],
+  "references": [
```

Comment:
Have you checked that cross package navigation, content assist and so on in VS Code and Theia works well?

Reply:
I did not have issues on VS Code, I can check with Theia.